packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1496,6 +1497,8 @@ export class GarbageCollector implements IGarbageCollector {
                 : this.sweepTimeoutMs,
             completedGCRuns: this.completedRuns,
             lastSummaryTime: this.getLastSummaryTimestampMs(),
+            containerCreateTime: this.metadata?.createContainerTimestamp,
+            containerRuntimeVersion: this.metadata?.createContainerRuntimeVersion,
```

Comment:
make sure this matches the existing column name logged in the "ContainerCreateStats" event (I think that's what it's called).  I think the name is different there but could be remembering wrong.